lib/extlib-abc/aig/cnf/cnfData.c

```diff
@@ -4557,7 +4557,7 @@ void Cnf_ReadMsops( char ** ppSopSizes, char *** ppSops )
         { 0x0F0F, 0xF0F0 },
         { 0x00FF, 0xFF00 }
     };
-    signed char Map[256], * pPrev, * pMemory;
+    char Map[256], * pPrev, * pMemory;
```

Comment:
Could you elaborate on the removal of `signed`? I believe in C the signedness of the `char` type is architecture defined. On ARM for instance it is unsigned by default, so this could change the behavior on ARM systems.

Reply:
Yes. I mention this in the commit message. PowerPC is one of the architectures that makes the unusual choice of having char unsigned by default.

I removed the signed because clang (on an x86_64 machine) was complaining about mixing pointers to signed and unsigned chars. Specifically
```
[ELIDED]/lib/extlib-abc/aig/cnf/cnfData.c:4594:24: warning: assigning to
      'char *' from 'signed char *' converts between pointers to integer types with different sign
      [-Wpointer-sign]
            pSops[i++] = pPrev;
                       ^ ~~~~~
```